### PR TITLE
fix: remove references to external typings

### PR DESCRIPTION
**Pull request by piotr-oles**
Fixes E2E test to ensure that the package typings can be consumed without need to install external packages except `@types/webpack`

Solves: #339

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -349,6 +349,20 @@ hooks.waiting.tap('yourListenerName', () => {
 });
 ```
 
+## Typings
+
+To use the plugin typings, you have to install `@types/webpack`. It's not included by default to not collide with your
+existing typings (`@types/webpack` imports `@types/node`). [It's an old TypeScript issue](https://github.com/microsoft/TypeScript/issues/18588), 
+the alternative is to set `skipLibCheck: true` in the `compilerOptions` 😉
```
Comment (johnnyreilly) [inline README.md:356]: Did you know that `skipLibCheck: true` is the default as of TypeScript 3.9?

https://www.twitter.com/orta/status/1260583041740812288

**Review by johnnyreilly — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (piotr-oles) [reply README.md:356]: Good to know 😃

**Pull request merged**